src/assets/main.css

```diff
@@ -3,7 +3,7 @@
 :root {
     --gryphon-red: #C20430;
     --gryphon-yellow: #FFC72A;
-    --gryphon-black: #000000;
+    --gryphon-black: #110f0fee;
```

Comment:
This doesn't follow the university style guide, if you want to use a different color, make a different variable